packages/abstract-signer/src.ts/index.ts

```diff
@@ -29,7 +29,7 @@ const logger = new Logger(version);
 
 const allowedTransactionKeys: Array<string> = [
     "accessList", "chainId", "customData", "data", "from", "gasLimit", "maxFeePerGas", "maxPriorityFeePerGas", "to", "type", "value",
-    "nodeId"
+    "nodeId", "isSimpleTransfer"
```

Comment:
This should be removed when we move the `isSimpleTransfer` to the `customData` field.